### DISCUSSION TITLE
Is bb's future development dead?

### DISCUSSION THREAD

**Discussion by Unnameless** [Chia-Network/bladebit#471] [General]
Hello everyone,

I'm interested to know, preferable  @harold-b , or anyone else knowledgeable on the matter, if the future development for higher compression levels is dead or stagnated!

On the mainpage of this project it states that:

`NOTE:`` Although higher compression levels are available, support for farming them has not been currently implemented and are therefore disabled. They will be implemented in the future.`

However the last version of bladebit has been released almost 1 year ago. Case in which I would like to know what's going on with development of the higher compression levels for both plotting and farming.

Thank you

**Comment by haorldbchi** [comment]
Hello, this is @harold-b (from my work account).

For a while now we've diverted our resources to focus on the new plot format. I am part of the team that is working on it, and as such I've been completely devoted to that endeavor. It's wiser at this point to ensure we deliver a good replacement format instead of dedicated more resources to a format that will be deprecated in the future.

Nick (Dr. Plotter) is writing a blog post elaborating more on the new plot format that should be published soon, if you're interested.

Thanks for your interest and asking the question

**Reply by Unnameless** [reply]
Thank you for the swift and kind reply!

Will the new format still be open-source? Regardless of the answer, is it a bad idea to have the software with a paid option or some sort of reward. I mean shouldn't your work be rewarded in some way?

**Reply by haorldbchi** [reply]
It will be open source and free as always. There's no rewarded or closed-source option.

**Reply by Unnameless** [reply]
Thank you!

**Comment by hajes** [comment]
there is a new format coming after acquisition of Dr. Plotter

**Reply by Unnameless** [reply]
what do you mean by "acquisition" ?

**Reply by haorldbchi** [reply]
The new plot format is completely unrelated to the current DrPlotter software, it has nothing to do with it. The only relation is that Nick, who wrote it, is a core part of the team working on, and designing it
